logwarn_openqa

```diff
@@ -86,5 +86,7 @@ $logwarn ${options} -m '\[.*:?(debug|info|warn|error)\]' -p ${file} \
     '!\[error\].*naive_verify_failed_return' \
     `# https://progress.opensuse.org/issues/138536` \
     '!\[error\].*Worker [0-9]+ has no heartbeat.*' \
+    `# https://progress.opensuse.org/issues/178147` \
+    `!\[error\] \[pid:[0-9]+\] Publishing opensuse\.openqa\.comment\.create failed: Can\'t connect: System error \(.+job ID: [0-9]+, 9 attempts left\)` \
```

Comment:
should the 9 appear here?

Reply:
Apparently it's not considered necessary